packages/block-library/src/query/edit/inspector-controls/index.js

```diff
@@ -128,11 +130,18 @@ export default function QueryInspectorControls( props ) {
 		isControlAllowed( allowedControls, 'parents' ) &&
 		isPostTypeHierarchical;
 
+	// TODO: this condition does not work for custom post types,
+	// useTaxonomies util does not return the post format taxonomy
+	// for custom post types, only posts?
+	const showPostFormatControl =
```

Comment:
Have you tested by adding the support in the custom post type? By reading this it seems it's not supported by default..

Reply:
Yes, I registered the support. The post format shows in the format option in the block editor.

Reply:
I have replaced this with a check for the theme support.